dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/relationship/hibernate/RelationshipType.hbm.xml

```diff
@@ -21,6 +21,12 @@
 
         <property name="translations" type="jblTranslations"/>
 
+        <property name="bidirectional" column="bidirectional" not-null="true" />
+
+        <property name="fromToName" column="fromToName" not-null="true" />
```

Comment:
Didnt understand this fully. But if the relationshiptype is unidirectional, is the expectation that fromToName will be the same as toFromName and both will not be null ?

Reply:
No, only from-to name is required, since there is always atleast one name/direction. toFrom is the reverse and is optional and will be null for non-bidirectional relationships

Reply:
So the property below should be (not-null="false") ?

Reply:
yes :)